pkg/controller/servicebindingrequest/binding.go

```diff
@@ -60,26 +68,26 @@ type ServiceBinder struct {
 	Logger *log.Log
 	// Objects is a list of additional unstructured objects related to the Service Binding Request.
 	Objects []*unstructured.Unstructured
-	// SBR is the ServiceBindingRequest associated with binding..
+	// SBR is the ServiceBindingRequest associated with binding.
 	SBR *v1alpha1.ServiceBindingRequest
 	// Secret is the Secret associated with the Service Binding Request.
 	Secret *Secret
 }
 
-// updateServiceBindingRequest execute update API call on a SBR request. It can return errors from
-// this action.
-func (b *ServiceBinder) updateServiceBindingRequest(
+func UpdateServiceBindingRequest(
```

Comment:
This method needs a documentation comment

Reply:
Is this really need to be an exported method?

Reply:
yes, reconciler.go is using it

Reply:
reconciler.go is in the same package, right?